src/main/java/cs/f10/t1/nursetraverse/ui/Observer.java

```diff
@@ -0,0 +1,23 @@
+package cs.f10.t1.nursetraverse.ui;
+
+import javafx.scene.input.KeyCode;
+
+/**
+ * An observer interface to observe the command box class for userinputs
+ * This interface was created so as to provide a facade as ui interacts with one another
+ */
+public interface Observer {
```

Comment:
There's already an Observer class in Java. Might want to be more specific on what kind of observer this class is in the name.

Reply:
The observer class name seems abit too general. i have changed it to "'ObserverUi" instead